app/services/plans/update_service.rb

```diff
@@ -47,6 +47,7 @@ def call
         process_charges(plan, params[:charges]) if params[:charges]
         process_minimum_commitment(plan, params[:minimum_commitment]) if params[:minimum_commitment] && License.premium?
         process_downgraded_subscriptions if old_amount_cents != plan.amount_cents
+        process_pending_subscriptions if old_amount_cents != plan.amount_cents
```

Comment:
Since the condition is the same, maybe you could group them:
```suggestion
        if old_amount_cents != plan.amount_cents
          process_downgraded_subscriptions
          process_pending_subscriptions
        end
```